src/components/visible-while-frozen.js

```diff
@@ -36,10 +36,18 @@ AFRAME.registerComponent("visible-while-frozen", {
     const isFrozen = this.el.sceneEl.is("frozen");
 
     let isWithinDistance = true;
+    const isVisible = this.el.getAttribute("visible");
```

Comment:
We try not to use these fake components anymore:
```suggestion
    const isVisible = this.el.object3D.visible;
```